Casks/wireshark.rb

```diff
@@ -18,15 +18,20 @@
   uninstall_preflight do
     set_ownership '/Library/Application Support/Wireshark'
 
-    if File.read('/etc/group').match?(%r{^access_bpf})
+    if system_command '/usr/bin/dscl', args: ['.', '-read', '/Groups/access_bpf'], must_succeed: false
       system_command '/usr/sbin/dseditgroup',
-                     args: ['-o', 'delete', 'access_bpf'],
+                     args: ['-q', '-o', 'delete', 'access_bpf'],
                      sudo: true
     end
   end
 
   uninstall pkgutil:   'org.wireshark.*',
             launchctl: 'org.wireshark.ChmodBPF'
 
-  zap trash: '~/Library/Saved Application State/org.wireshark.Wireshark.savedState'
+  zap trash: [
+               '~/Library/Saved Application State/org.wireshark.Wireshark.savedState',
+               '~/Library/Caches/org.wireshark.Wireshark',
+               '~/Library/Cookies/org.wireshark.Wireshark.binarycookies',
+               '~/Library/Preferences/org.wireshark.Wireshark.plist',
+             ]
```

Comment:
This is really a cosmetic thing but helps maintainers later on, sort those alphabetically: 
```suggestion
  zap trash: [
               '~/Library/Caches/org.wireshark.Wireshark',
               '~/Library/Cookies/org.wireshark.Wireshark.binarycookies',
               '~/Library/Preferences/org.wireshark.Wireshark.plist',
               '~/Library/Saved Application State/org.wireshark.Wireshark.savedState',
             ]
```